src/js/helpers/JSONPUtil.js

```diff
@@ -0,0 +1,42 @@
+const JSONPUtil = {
+  /**
+   * Request JSONP data
+   *
+   * @todo: write test to verify this util is working properly
+   *
+   * @param {string} url
+   * @returns {Promise} promise
+   */
```

Comment:
Aight, I'll remove the comments. :smile: As these comments add value in documenting parameter/return types we might consider add such comments to all of our modules.